framework/python/src/core/testrun.py

```diff
@@ -463,7 +463,7 @@ def start_ui(self):
 
     try:
       client.containers.run(
-            image='test-run/ui',
+            image='testrun/ui',
             auto_remove=True,

```

Comment:
Why change this format?  All other images use test-trun as naming convention.

Reply:
Rolling the change out slowly. Eventually they will all be changed to testrun as that is the correct naming.